pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
Enabling delete permission or removing nfsv4 acl may not be an acceptable solution. NFS servers, e.g. from NetApp, set NFS v4 ACL on a per controller basis [here](https://library.netapp.com/ecm/ecm_download_file/ECMP1196891) in page 98 (proofreading appreciated). 

That leaves us with NFS v3 option. If we drop NFS v4 support, would that be acceptable? @liggitt  @pweil- @markturansky @smarterclayton